etl/pom.xml

```diff
@@ -61,38 +61,6 @@
             <version>1.78</version>
         </dependency>
 
-
-        <dependency>
-            <groupId>pentaho-kettle</groupId>
-            <artifactId>kettle-core</artifactId>
-            <version>7.1-SNAPSHOT</version>
-            <exclusions>
-                <exclusion>
-                    <artifactId>xercesImpl</artifactId>
-                    <groupId>xerces</groupId>
-                </exclusion>
-            </exclusions>
-            <scope>compile</scope>
-        </dependency>
-        <dependency>
-            <groupId>pentaho-kettle</groupId>
-            <artifactId>kettle-engine</artifactId>
-            <version>7.1-SNAPSHOT</version>
-            <scope>compile</scope>
-        </dependency>
-        <dependency>
```

Comment:
From the commits, I *think* Rhino and javax.mail were only used by the Pentaho code?